content/automate/ManageIQ/Transformation/Common.class/__methods__/acquiretransformationhost.rb

```diff
@@ -0,0 +1,49 @@
+module ManageIQ
+  module Automate
+    module Transformation
+      module Common
+        class AcquireTransformationHost
+          def initialize(handle = $evm)
+            @handle = handle
+          end
+
+          def main
+            begin
+              factory_config = @handle.get_state_var(:factory_config)
+              raise "No factory config found. Aborting." if factory_config.nil?
+
+              task = @handle.root['service_template_transformation_plan_task']
+              @handle.log(:info, "Task: #{task.inspect}")
+              source_vm = task.source
+              @handle.log(:info, "Source VM: #{source_vm.name}")
+              source_cluster = source_vm.ems_cluster
+              @handle.log(:info, "Source Cluster: #{source_cluster.name}")
```

Comment:
@fdupont-redhat 
You dont need the .name
You can just change it to 
```
@handle.log(:info, "Source Cluster: #{source_cluster}")
```
By default the to_s method that is indirectly called here will look for an attribute called name.
https://github.com/ManageIQ/manageiq-automation_engine/blob/a05c976f0983e18aea96f5d9609c931f75d7ad31/lib/miq_automation_engine/engine/miq_ae_method_service/miq_ae_service_object.rb#L20

Reply:
So, that will be the same anytime I use `.name`, isn't it ?

Reply:
@fdupont-redhat 
You dont have to make this change, if you just used #{vmdb_object} you will get the name attribute.